subsys/bluetooth/controller/ll_sw/ull_peripheral_iso.c

```diff
@@ -268,6 +268,22 @@ uint8_t ull_peripheral_iso_acquire(struct ll_conn *acl,
 	cis->lll.tx.payload_count = 0;
 	cis->lll.tx.bn_curr = 1U;
 
+	/* Set the flush points in nr. of subevents */
+	/* TODO: verify that the use of DIV_ROUND_UP is the right thing to do */
+	if (cis->lll.tx.bn != 0) {
+		cis->lll.tx.ft_cntr_se = cis->lll.tx.ft *
+			DIV_ROUND_UP(cis->lll.nse, cis->lll.tx.bn);
+	} else {
+		cis->lll.tx.ft_cntr_se = 0;
+	}
+	if (cis->lll.rx.bn != 0) {
+		cis->lll.rx.ft_cntr_se = cis->lll.rx.ft *
+			DIV_ROUND_UP(cis->lll.nse, cis->lll.rx.bn);
+	} else {
+		cis->lll.rx.ft_cntr_se = 0;
+	}
+
+
```

Comment:
We use the following for peripheral, if it's any help. based on BT Core 5.2, Vol 6, Part B, section 4.5.13.5:
E = floor(cisPayloadNumber / BN)
U = NSE - floor(NSE / BN) * (BN - 1 - (cisPayloadNumber mod BN))

E:  (E + FT - 1) = CIS event where flush point occurs
U:  Flush point sub event
FE: Flush point event
```
flush_event = (payload_number / rxtx->bn) + rxtx->ft - 1;
flush_subevents = nse - (nse / rxtx->bn) * (rxtx->bn - 1 - (payload_number % rxtx->bn));
```
                       
`flush_event`  is the event count at which the payload is flushed
`flush_subevents` is the number of subevents into `flush_event` where flush point occurs

This passes qualification.

Reply:
Thanks, I will  make sure that we follow the same